list.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-var github = require('github-repositories')
 var Promise = require('promise')
+var github = Promise.denodeify(require('github-repositories'))
```

Comment:
I understand, but Node.js 0.10 isn't going away any time soon, and the jstransformer core is already using the denodeify function anyway. Plus this module isn't supposed to be Browserified either so we don't really have a reason not to use the extensions provided by the module.